operations/boot_orchestration/Ascertain_Which_BOS_Session_Booted_A_Node.md

```diff
@@ -0,0 +1,72 @@
+# Ascertain Which BOS Session Booted a Node
+
+This guide is split into a BOS Version 1 (V1) and BOS Version 2 (V2) section
+because the procedures are different for the two different versions.
+
+## BOS Version 1 (V1)
+
+To determine which BOS Session booted or rebooted a node, query the node's
+kernel boot parameters. They contain a string 'bos_session_id' which indentifies
+which BOS session booted or rebooted the node. Then, use this BOS Session ID
+to describe the BOS Session, which identifies the BOS Session template used.
+
+### Query the Node
+
+From a management node (master or worker), ssh to the node in question.
+(`ncn-mw#`) ssh <node's xname>
+('<node>#') cat /proc/cmdline
+
+Find the bos_session_id value in the cmdline string. This is the ID of the BOS
+session that was used to boot the node.
+
+Exit from the node.
+('<node>#') exit
+
+### Query BOS
+Ask BOS to describe this session.
+(`ncn-mw#`#) cray bos v1 session describe  <BOS session ID> --format json
+
+The templateName parameter is the BOS session template used to boot or reboot the node.
```

Comment:
Why document this extra step?  The document is about determining the session, not the session template, and you don't do this in the V2 section.

Reply:
I _did_ describe the V2 session in the V2 section.

Reply:
I added the session template description output to make it more symmetric.